### PR TITLE
Adds logic to account for odd numbered columns

### PR DESCRIPTION
**Pull request by maccesar21**
Math was a bit off and was having some issues with odd numbered columns.

Logic should now account for that as well as style columns width based on total number of columns.

### CODE REVIEW

File: src/header/css/header-menus.pcss [hunk 1]
```diff
@@ -446,7 +446,23 @@
 				&[data-structuredcols=""] {
 					width: calc(300px * 2 + 20px + 35px * 2);
 					&[data-hasMultiCols="true"] {
-						width: calc(290px * 3 + 20px + 20px * 3);
+						&[data-filled="12"],
```
Comment (paulcpederson) [inline src/header/css/header-menus.pcss]: this is getting a little wacky. I would maybe just pass a modifier css class to these columns from JS?

Comment (maccesar21) [reply src/header/css/header-menus.pcss]: @paulcpederson updated this logic.. should be cleaner.

File: src/header/js/header-menus.js [hunk 2]
```diff
@@ -230,27 +240,23 @@ export default ({variant = 'desktop'}) => {
...
-			}
+		if (item.menus.length >= 10 && item.menus.length <= 18) {
+			const multi = Math.ceil(item.menus.length / 2);
+			columns = $('div', {class: `${prefix}-sublist--col-wrapper columns2`},
+				createMenuColumns(item.menus.slice(0, multi)),
+				createMenuColumns(item.menus.slice(multi, item.menus.length))
+			);
+		} else if (item.menus.length > 18 && item.menus.length <= 27) {
+			const multi = Math.ceil(item.menus.length / 3);
+			columns = $('div', {class: `${prefix}-sublist--col-wrapper columns3`},
```
Comment (paulcpederson) [inline src/header/js/header-menus.js]: using the class `columns3` is kind of risky as it's so generic. Would be better to do something like `${prefix}-columns-3` to namespace it.

Comment (maccesar21) [reply src/header/js/header-menus.js]: @paulcpederson made those updates.. Good call!

**Pull request merged**